packages/inference/test/HfInference.spec.ts

```diff
@@ -645,11 +639,89 @@ describe.concurrent(
 		});
 
 		it("endpoint - makes request to specified endpoint", async () => {
-			const ep = hf.endpoint("https://api-inference.huggingface.co/models/google/flan-t5-xxl");
+			const ep = hf.endpoint("https://api-inference.huggingface.co/models/openai-community/gpt2");
 			const { generated_text } = await ep.textGeneration({
 				inputs: "one plus two equals",
 			});
-			expect(generated_text).toEqual("three");
+			assert.include(generated_text, "three");
+		});
+
+		it("chatCompletion - OpenAI Specs", async () => {
+			const ep = hf.endpoint(
+				"https://api-inference.huggingface.co/models/mistralai/Mistral-7B-Instruct-v0.2/v1/chat/completions"
+			);
+			const res = await ep.chatCompletion({
+				model: "tgi",
+				messages: [{ role: "user", content: "Complete the this sentence with words one plus one is equal " }],
+				max_tokens: 500,
+				return_full_text: false,
+				temperature: 0.1,
+				seed: 0,
+			});
+			if (res.choices && res.choices.length > 0) {
+				const completion = res.choices[0].message?.content;
+				expect(completion).toContain("to two");
+			}
+		});
+		it("chatCompletionStream - OpenAI Specs", async () => {
+			const ep = hf.endpoint(
+				"https://api-inference.huggingface.co/models/mistralai/Mistral-7B-Instruct-v0.2/v1/chat/completions"
+			);
+			const stream = ep.chatCompletionStream({
+				model: "tgi",
+				messages: [{ role: "user", content: "Complete the equation 1+1= ,just the answer" }],
+				max_tokens: 500,
+				return_full_text: false,
+				temperature: 0.1,
+				seed: 0,
+			});
+			let out = "";
+			for await (const chunk of stream) {
+				if (chunk.choices && chunk.choices.length > 0) {
+					out += chunk.choices[0].delta.content;
+				}
+			}
+			expect(out).toContain("2");
+		});
+		it("custom mistral - OpenAI Specs", async () => {
+			const MISTRAL_KEY = env.MISTRAL_KEY;
+			if (!MISTRAL_KEY) {
+				console.warn("Skipping test because MISTRAL_KEY is not set");
+				return;
+			}
```

Comment:
@radames can you remove those conditional skips?

You can do `VCR_MODE=cache` in the env before running the test to record the result in `tapes.json`.

Reply:
done